qdrant_client/qdrant_client.py

```diff
@@ -1240,6 +1241,9 @@ def set_payload(
                 If multiple shard_keys are provided, the update will be written to each of them.
                 Only works for collections with `custom` sharding method.
 
+            key:
+                Assigns payload to each point that satisfy this path of property
```

Comment:
Will try to suggest better comment...

Reply:
fair point.. I just copied it from openapi 🙃